async-http-client-backend/zio/src/main/scala/sttp/client3/asynchttpclient/zio/AsyncHttpClientZioBackend.scala

```diff
@@ -251,6 +251,7 @@ object AsyncHttpClientZioBackend {
     *
     * See [[SttpBackendStub]] for details on how to configure stub responses.
     */
-  def stub: SttpBackendStub[Task, ZioStreams with WebSockets] =
+  def stub[T]: SttpBackendStub[Task, T] =
```

Comment:
why the type parameter here? the stub should support the same capabilities as the backend?

Reply:
Without the type parameter, there will be an error: sttp/async-http-client-backend/zio/src/test/scala/sttp/client3/asynchttpclient/zio/SttpBackendStubZioTests.scala:75:12: Cannot prove that sttp.capabilities.zio.ZioStreams with sttp.capabilities.WebSockets with sttp.capabilities.Effect[[+A]zio.Task[A]] <:< Any with sttp.capabilities.Effect[zio.Task] with sttp.client3.testing.TestStreams.
I will investigate how to fix this issue whenever I get a chance.

Reply:
Removed the type parameter.